cardano_node_tests/tests/test_plutus_spend_build.py

```diff
@@ -195,6 +198,32 @@ def payment_addrs_lock_always_fails(
     return addrs
 
 
+@pytest.fixture
+def fund_script_guessing_game(
+    cluster: clusterlib.ClusterLib,
```

Comment:
This need to use `cluster_lock_guessing_game`, to make sure no other test that uses the "guessing game" plutus script run at the same time.